packages/utils/Logger.ts

```diff
@@ -26,11 +25,7 @@ interface TargetOptions {
 	path?: string;
 	source?: string;
 
-	// Default message format
-	format?: string;
-
-	// If specified, will use this as format if it's an info message
-	formatInfo?: string;
+	formatter?: (level: LogLevel, targetPrefix?: string)=> string;
```

Comment:
Could you keep the name `format` please, and it would be either a function (new behaviour) or a string (old behaviour)